speculos.py

```diff
@@ -128,29 +160,40 @@ def run_qemu(s1, s2, app_path, libraries=[], seed=DEFAULT_SEED, debug=False, tra
     args = parser.parse_args()
     args.model.lower()
 
-    if not (args.flushed or args.progressive):
-        if args.model == 'blue':
-            args.progressive = True
+    try:
```

Comment:
I wouldn't put everything under try/except because part of this code don't raise exceptions. Also I would directly exit with an error message instead of raising `ValueError`

Reply:
You don't like the exception mechanism , do you? ;-)
- "try" clause moved closer to where exceptions could be raised (e.g. run_qemu() which uses functions from "os" module which in turn can raise a FileNotFound exception for whatever reason). 
- Replaced the "raise ValueError" statements w/ print()/exit() code.

Reply:
Actually I don't see the point of catching an exception if no action is made but writing the error message and exiting :)

Reply:
This was to make the error printing cleaner rather than having the full exception  stack dumped on the user in case of network or file error. But we can live w/ exception stack, right?
=> removed the try/except clause entirely.